### PR TITLE
anuragwiki

### PR DESCRIPTION
**Pull request by anurag629**
created notification app and fixed all imports

### CODE REVIEW

**Comment by anurag629** [comment]
Sir please review my PR.
Tell me what i have done wrong in this PR

File: project/accounts/migrations/0005_alter_profile_categories.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
+# Generated by Django 3.2.7 on 2021-09-28 08:43
+=======
 # Generated by Django 3.2.7 on 2021-09-27 09:25
+>>>>>>> f693e3c878920b2ff58c282b4f88f507b61bf985
```
Comment (brylie) [inline project/accounts/migrations/0005_alter_profile_categories.py]:
```suggestion
# Generated by Django 3.2.7 on 2021-09-28 08:43
```

File: project/api/serializers.py [hunk 2]
```diff
@@ -1,6 +1,12 @@
 from rest_framework import serializers
 
+<<<<<<< HEAD
+from accounts.forms import UpdateProfileImage
 from api.models import Civi, Thread, CiviImage, Activity
+from categories.models import Category
+=======
+from api.models import Civi, Thread, CiviImage, Activity
+>>>>>>> f693e3c878920b2ff58c282b4f88f507b61bf985
```
Comment (brylie) [inline project/api/serializers.py]:
```suggestion
from accounts.forms import UpdateProfileImage
from api.models import Civi, Thread, CiviImage, Activity
from categories.models import Category
```

File: project/api/views/category.py [hunk 3]
```diff
@@ -3,6 +3,10 @@
 from rest_framework.response import Response
 
 from api.models import Thread
+<<<<<<< HEAD
+from categories.models import Category
+=======
+>>>>>>> f693e3c878920b2ff58c282b4f88f507b61bf985
```
Comment (brylie) [inline project/api/views/category.py]:
```suggestion
from categories.models import Category
```

File: project/api/write.py [hunk 4]
```diff
@@ -18,8 +18,15 @@
...
 # civi packages
+<<<<<<< HEAD
+from accounts.forms import UpdateProfileImage
 from api.models import Thread
 from .models import Activity, Civi, CiviImage
+from categories.models import Category
+=======
+from api.models import Thread
+from .models import Activity, Civi, CiviImage
+>>>>>>> f693e3c878920b2ff58c282b4f88f507b61bf985
```
Comment (brylie) [inline project/api/write.py]:
```suggestion
from accounts.forms import UpdateProfileImage
from api.models import Thread
from .models import Activity, Civi, CiviImage
from categories.models import Category
```

File: project/categories/migrations/0001_initial.py [hunk 5]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
+# Generated by Django 3.2.7 on 2021-09-28 08:43
+=======
 # Generated by Django 3.2.7 on 2021-09-27 09:25
+>>>>>>> f693e3c878920b2ff58c282b4f88f507b61bf985
```
Comment (brylie) [inline project/categories/migrations/0001_initial.py]:
```suggestion
# Generated by Django 3.2.7 on 2021-09-28 08:43
```

File: project/frontend_views/views.py [hunk 6]
```diff
@@ -7,9 +7,16 @@
...
+<<<<<<< HEAD
+from categories.models import Category
+from accounts.models import Profile
+from accounts.forms import ProfileEditForm
+from accounts.forms import UpdateProfileImage
+=======
 from accounts.models import Profile
 from accounts.forms import ProfileEditForm, UpdateProfileImage
 from categories.models import Category
+>>>>>>> f693e3c878920b2ff58c282b4f88f507b61bf985
```
Comment (brylie) [inline project/frontend_views/views.py]:
```suggestion
from accounts.models import Profile
from accounts.forms import ProfileEditForm, UpdateProfileImage
from categories.models import Category
```

**Review by brylie — CHANGES_REQUESTED** [review]
Remove/resolve merge comments

**Comment by brylie** [comment]
Closing this pull request, since resolving conflicts required a rebase. I opened pull request #1055 with your changes @anurag629 and you will still get credit for the work :smiley:

**Comment by anurag629** [comment]
what should i do now sir ?

**Comment by gorkemarslan** [comment]
@anurag629, your changes are moved to PR #1055. Could you continue with that PR?

**Comment by brylie** [comment]
Perhaps we should merge #1055?

**Pull request closed without merging**